src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/register")
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/183255986-8203aba9-e7a6-4d0e-a261-34ea83755913.png)
Before you create PR with your homework, you need to go through the description of the task and checklist under the task, and correct all the points described there. The mentor will not check the work until the checklist items are corrected